docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.7"
+services:
+  elastic:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.3
+    container_name: elastic
+    environment:
+      - node.name=elastic
+      - discovery.type=single-node
+      - cluster.name=docker-cluster
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - elastic:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
+    networks:
+      - backend
+    restart: on-failure
+
+  natlas-server:
+    image: natlas-server
+    ports:
+      - 5000:5000
+    networks:
+      - backend
+    environment:
+      - ELASTICSEARCH_URL=http://elastic:9200
+      - BIND_ADDRESS=0.0.0.0
```

Comment:
What is this bind address for? It's not a supported environment variable by natlas or gunicorn as far as I can tell. I'm also extremely hesitant to ship any files that would recommend putting the flask application directly on any network, especially the internet.

Reply:
I'm going to put nginx in front of this and this actually has no purpose since its already 0.0.0.0 in the run command.

Reply:
Oh right, 0.0.0.0 inside a docker container is not the same as 0.0.0.0 on the host. Either way, if it doesn't serve a purpose then let's clean it up :)